process/constants.go

```diff
@@ -84,7 +84,7 @@ const MinForkRound = uint64(0)
 
 // MaxNumPendingMiniBlocksPerShard defines the maximum number of pending miniblocks per shard, after which a shard
 // could be considered stuck if this value multiplied by the number of shards is reached
-const MaxNumPendingMiniBlocksPerShard = 30
+const MaxNumPendingMiniBlocksPerShard = 60
```

Comment:
since this is strictly related to the maximum number of miniblocks resulted by the split, I suggest moving this also in the economics.toml file or somehow automatically computed by splitting the max gas limit per block with the current miniblock gas limit multiplied by a const like 10.

Reply:
I think the second suggestion is better